clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -382,7 +382,8 @@ Changes in existing checks
   using pointer to member function. Additionally, the check no longer emits
   a diagnostic when a variable that is not type-dependent is an operand of a
   type-dependent binary operator. Improved performance of the check through
-  optimizations.
+  optimizations. The check no longer emits a diagnostic for
+  variables used as the initializer of C++17 fold expressions.
```

Comment:
I am not sure "initializer of C++17 fold expressions" is the correct description for `InitExpr` in fold expression since I don't find it in https://eel.is/c++draft/expr.prim.fold and https://en.cppreference.com/w/cpp/language/fold.

Reply:
You are right, what do you think about `The check no longer emits a diagnostic for non-parameter-pack variables in C++17 fold expressions.`?

Reply:
Done